install/mlops/truera/install.yml

```diff
@@ -0,0 +1,16 @@
+id: truera-mlops
+name: Truera Inc
+
+title: Truera diagnositcs and monitoring solution for Trustworthy AI.
```

Comment:
Correct the typo in `diagnositics` > `diagnostics`.
You can also maintain more consistency by keeping the lowercase on `trustworthy`.